hicommonwealth/commonwealth
# Added custom domain functions to admin panel

## Link to Issue
Closes: #8226
Closes: #7683 

## Description of Changes
- Adds ability to add and update new custom domains on admin panel

## Test Plan
- Add HEROKU_APP_NAME, HEROKU_API_TOKEN, MAGIC_API_KEY, and MAGIC_CLIENT_ID to .env
- Go to http://localhost:8080/admin panel and make sure you are logged in as a super admin.
- Refresh existing custom domain and ensure it returns data
- Try refreshing a non-existing custom domain and ensure it returns error
- Try adding a new custom domain to a community. Then try refreshing it to ensure it returns data.  
- Try adding the same custom domain twice, make sure it throws an error

## Deployment plan
- We need to add MAGIC_CLIENT_ID to the environment variables as it is not there currently
